protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -501,8 +504,25 @@ func (k Keeper) internalCanUpdateSubaccounts(
 			)
 		}
 
-		if exists && currentBlock-lastBlockNegativeTncSubaccountSeen <
-			types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS {
+		// Panic if the current block is less than the last block a chain outage was seen.
+		downtimeInfo := k.blocktimeKeeper.GetDowntimeInfoFor(ctx, 5*time.Minute)
+		if currentBlock < downtimeInfo.BlockInfo.Height {
+			panic(
+				fmt.Sprintf(
+					"internalCanUpdateSubaccounts: current block (%d) is less than the last "+
+						"block a chain outage was seen (%d)",
+					currentBlock,
+					downtimeInfo.BlockInfo.Height,
+				),
+			)
+		}
+
+		negativeTncSubaccountSeen := exists && currentBlock-lastBlockNegativeTncSubaccountSeen <
+			types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS
+		chainOutageSeen := downtimeInfo.BlockInfo.Height > 0 && currentBlock-downtimeInfo.BlockInfo.Height <
+			types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS
```

Comment:
Should we be using the same constant value for both last time a negative tnc subaccount was seen and the last time a chain outage was seen?

Reply:
Yeah I think it's fine

idea is this gives enough time for an honest proposer to propose a block